LLVM 7 complains about local variable that will be copied despite being returned by name

When using cxxopts with clang/llvm 7 and `-Werror,-Wreturn-std-move`, I get the following error message:

> cxxopts/include/cxxopts.hpp:200:12: error: local variable 't' will be copied despite being returned by name

--- a/include/cxxopts.hpp
+++ b/include/cxxopts.hpp
@@ -210,7 +210,7 @@ namespace cxxopts
   T
   toLocalString(T&& t)
   {
-    return std::move(t);
+    return std::forward<T>(t);
   }
 
   inline

--- a/include/cxxopts.hpp
+++ b/include/cxxopts.hpp
@@ -210,7 +210,7 @@ namespace cxxopts
   T
   toLocalString(T&& t)
   {
-    return t;
+    return std::move(t);
   }
 
   inline

